05_syntax_lab/01.py

```diff
@@ -1,17 +1,17 @@
 #Max Number
+import sys

```

Comment:
Nice. Now try without using ```sys```.
* You don't need to set a maximum outside the loop
* ```raw_input``` can have a parameter